Engine/Annotations/Annotation.cs

```diff
@@ -1316,17 +1317,22 @@ public IEnumerable Selected
             }
         }
 
-
         class InputStepAnnotation : IAvailableValuesSelectedAnnotation, IOwnedAnnotation, IStringReadOnlyValueAnnotation
         {
+            public class Strings : StringLocalizer<Strings>
+            {
+                public readonly string None = "None";
```

Comment:
Can we do ```public string None => GetTranslation("None");```? I think that would be a bit less magical. 

Also, a translator can then be used to get all the existing translation by calling the property getters through reflection.